features/i.yml

```diff
@@ -1,7 +1,7 @@
-draft_date: 2024-06-05
-name: <i>
+name: Italic element
 description: The `<i>` element TODO.
 spec: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element
 group: html
 compat_features:
   - html.elements.i
+  - html.elements.em
```

Comment:
Same here. I think we'll get an earful if folks find out we're combining `<i>` and `<em>`. 😅